tree/30_generic_methods/service_ensure_started_at_boot.cf

```diff
@@ -41,7 +41,7 @@ bundle agent service_ensure_started_at_boot(service_name)
       "args"                     slist => { "${service_name}" };
 
   methods:
-    "action"             usebundle => service_stopped("${service_name}");
+    "action"             usebundle => service_started("${service_name}");
```

Comment:
service_enabled

Reply:
commit modified.
but now we have a duplicate between service_enabled and service_ensure_started_at_boot; how is  that useful ?